sync/sync.go

```diff
@@ -294,15 +297,22 @@ func (sync *synchronizer) prepareBundle(msg message.Message) *bundle.Bundle {
 	return nil
 }
 
-func (sync *synchronizer) sendTo(msg message.Message, to peer.ID) {
+func (sync *synchronizer) sendTo(msg message.Message, to peer.ID, sessionID int) {
 	bdl := sync.prepareBundle(msg)
 	if bdl != nil {
 		data, _ := bdl.Encode()
 		err := sync.network.SendTo(data, to)
 		if err != nil {
 			sync.logger.Error("error on sending bundle", "bundle", bdl, "err", err, "to", to)
+			sync.peerSet.IncreaseSendFailedCounter(to)
```

Comment:
Should we increase the failed attempts when the connection is OK but request rejectes?

Reply:
It's done